<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 24pt &quot;微软雅黑&quot;;
background-color: rgb(0, 85, 127);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="recognize">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>310</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 127);
font: 20pt &quot;Segoe UI Semibold&quot;;
selection-background-color: rgb(85, 170, 127);
background-color: rgb(255, 255, 255);
border-color: rgb(0, 85, 127);
border-radius: 15px;
alternate-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
alternate-background-color: rgb(0, 85, 127);</string>
    </property>
    <property name="text">
     <string>Recognize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_gei">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>220</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 127);
font: 20pt &quot;Segoe UI Semibold&quot;;
selection-background-color: rgb(85, 170, 127);
background-color: rgb(255, 255, 255);
border-color: rgb(0, 85, 127);
border-radius: 15px;
alternate-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
alternate-background-color: rgb(0, 85, 127);</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="register_2">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>130</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Segoe UI Semibold&quot;;
color: rgb(0, 85, 127);
selection-background-color: rgb(85, 170, 127);
background-color: rgb(255, 255, 255);
border-radius: 15px;
border-color: rgb(0, 85, 127);
alternate-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
alternate-background-color: rgb(0, 85, 127);</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="id_name">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>600</y>
      <width>221</width>
      <height>57</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Segoe UI Semibold&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="id_num">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>600</y>
      <width>151</width>
      <height>57</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Segoe UI Semibold&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updater">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>400</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 127);
font: 20pt &quot;Segoe UI Semibold&quot;;
selection-background-color: rgb(85, 170, 127);
background-color: rgb(255, 255, 255);
border-color: rgb(0, 85, 127);
border-radius: 15px;
alternate-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
alternate-background-color: rgb(0, 85, 127);</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>501</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 28pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>Gait Recognition</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>600</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>Number:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>600</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>600</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>State:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
